src/server.py

```diff
@@ -55,6 +55,9 @@ def get_article(url, config = Config()):
     # uncomment this if 200 is desired in case of bad url
     # article.set_html(article.html if article.html else '<html></html>')
     article.parse()
+    if article.text == "":
```

Comment:
Is there any use case where its needed? As I understood from docs it should be used for domains (like cnn.com) rather then article links.

Reply:
My comment is about `build(url, memoize_articles=False)`

Reply:
since `build(url, memoize_articles=False)` return definition containing `description` ( which Article fails to get). `description` will return content in <head> <meta name="description">.

To sum up, this solution covers the case where text inside `body` is empty -> then we check `description` in header